boards/arm/quick_feather/Kconfig.defconfig

```diff
@@ -11,7 +11,7 @@ config BOARD
 if PWM
 
 config PWM_LITEX
-	select SOC_EOS_S3_FPGA
+	select EOS_S3_PROGRAM_FPGA
 	default y
 
```

Comment:
If you want to use your own driver instead of `PWM_LITEX` please add proper configs here like:
```suggestion
config PWM_EOS_S3
    select SOC_EOS_S3_FPGA
    select EOS_S3_PROGRAM_FPGA
    default y
```
and then disable `PWM_LITEX` in `prj.conf` in sample: `CONFIG_PWM_LITEX=n`